schemas/plugin.schema.json

```diff
@@ -0,0 +1,134 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "ID": {
+      "type": "string",
+      "oneOf": [
+        {
+          "pattern": "(?i)^[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$"
+        },
+        {
+          "pattern": "(?i)^[a-f0-9]{32}$"
+        }
+      ],
+      "description": "A unique identifier in the form of UUID so Flow Launcher can distinguish between plugins."
+    },
+    "ActionKeyword": {
```

Comment:
IIRC Actionkeyword is deprecated in favour of Actionkeywords. Right now they can be either, see the WebSearches plugin where it needs support of multiple action keywords, so might as well phase out `Actionkeyword` to avoid confusion.

Reply:
Done.